docs/job_types/job_golang.md

```diff
@@ -2,7 +2,7 @@
 "golang" job type is intended to handle the calls to your Go function.
 Racetrack will wrap it up in a web server.
 
-Set `lang: golang:latest` in your `job.yaml` manifest file in order to use this type of job.
+Set `jobtype: golang:latest` in your `job.yaml` manifest file in order to use this type of job.
```

Comment:
Let's get rid of this file. It's been moved to https://github.com/TheRacetrack/plugin-go-job-type/blob/master/docs/job_golang.md, but for some reason this leftover remained.